docker/dev/Dockerfile

```diff
@@ -89,9 +89,11 @@ RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" \
 
 # install debugger
 RUN \
+      if [ "$GOLANG_OS_ARCH" == "linux-amd64" ]; \
+      then \
 	go get -u github.com/go-delve/delve/cmd/dlv && dlv version; \
-	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && dep version;
-
+	curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh && dep version; \
```

Comment:
This is a call that installs dependency manager `dep`. Which actually could be replaced with `go get -u github.com/golang/dep/cmd/dep` and could work even for ARM.